lib/matplotlib/_mathtext.py

```diff
@@ -0,0 +1,2967 @@
+"""
+Implementation details for :mod:`.mathtext`.
+"""
+
+from collections import namedtuple
+import functools
+import logging
+from io import StringIO
```

Comment:
Not sure why these changed order.

Reply:
fixed, likely just a mistake

Reply:
It's still flipped, so I don't know why GitHub says it's outdated.

Reply:
let's do it again :/